src/internal/connector/onedrive/collection.go

```diff
@@ -64,12 +69,13 @@ type Collection struct {
 
 // itemReadFunc returns a reader for the specified item
 type itemReaderFunc func(
-	ctx context.Context,
+	hc *http.Client,
```

Comment:
Why remove `ctx`?

Reply:
Nevermind - I guess it's no longer used.

Reply:
yup